vanilla-installer/main.py

```diff
@@ -1,35 +1,37 @@
 """
 Most important functions of VanillaInstaller.
 """
-
-PATH_FILE = "data/mc-path.txt"
-FOLDER_LOC = ""
-TEMP_FOLDER = "temp/"
-
-import logging
-
 # IMPORTS
 import os
 import sys
-import zipfile
-import requests
+import logging
+import logging.handlers  # pylance moment
 import webbrowser
 import subprocess
+import pathlib
+import requests
 import minecraft_launcher_lib as mll
 
+
 # LOCAL
 import gui
 import theme
 
+PATH_FILE = str(pathlib.Path("data/mc-path.txt").resolve())
+FOLDER_LOC = ""
+TEMP_FOLDER = str(pathlib.Path("/temp").resolve())
```

Comment:
It will likely transform to `temp` folder in root directory, are you aware?

Reply:
Actually, that was probably meant to be a `temp/`.

Ideally there's an OS method for this somewhere, but I'll look into that.

Reply:
[This](https://docs.python.org/3/library/tempfile.html) maybe?

Reply:
Turned out that this is only used by `download_fabric()`, so this was easy to implement. Before this PR it was also used for `download_pack`, but that can't be used now because of Packwiz installer limitations.